packages/account/src/Sections/Security/LoginHistory/login-history-content.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { Table } from '@deriv/components';
+import { TLoginHistoryData } from '@deriv/api/types';
+import { fields } from '../../../../../utils/src/getLoginHistoryFormattedData';
+import LoginHistoryTableRow from './login-history-table-row';
+import LoginHistoryListRow from './login-history-list-row';
+
+type TGetFields = {
+    [key: string]: JSX.Element;
+};
+
+type TLoginHistoryContent = {
+    data: TLoginHistoryData;
+    is_mobile: boolean;
+};
+
+const getFields = () =>
+    Object.fromEntries(
+        Object.entries(fields).map(([key, value]) => [
+            key,
+            <Localize i18n_default_text='{{value}}' values={{ value }} key={key} />,
+        ])
+    );
+
+const LoginHistoryContent = ({ data, is_mobile }: TLoginHistoryContent) => {
```

Comment:
do we need this is_mobile to be passed as prop? cant we fetch is_mobile from ui-store in this component